ansible/configs/three-tier-app/requirements.yml

```diff
@@ -3,4 +3,4 @@
 
 - src: https://github.com/redhat-gpte-devopsautomation/ftl-injector
   name: ftl-injector
-  version: v0.11
+  version: 0.14.0
```

Comment:
scheme changed ?
`v0.14.0` vs `0.14.0` ?

Reply:
Yes, trying to be consistent across our portfolio. So since it isn't in production yet now was a good time to change.

That said if we, as a team, want to prefix with v happy to go that way. Thoughts re how semantic versioning should be applied?